template/.github/workflows/build.yml.j2

```diff
@@ -342,6 +346,9 @@ jobs:
       - name: Update version if PR
         if: ${{ github.event_name == 'pull_request' }}
         run: cargo set-version --offline --workspace 0.0.0-pr${{ github.event.pull_request.number }}
+      - name: Update version if manual trigger on non-main branch
```

Comment:
I think I would prefer if you could only run a workflow dispatch on `main`. Everything else introduces new image variants that currently don't exist (think of cleanup jobs and similar). Also we all have permissions to push an empty commit to all branches beside main (which also triggers ci)

Reply:
I agree on that. Afaik this pipeline runs on PR, thus this use case would be already covered I think.